src/scripts/main.js

```diff
@@ -20,9 +20,25 @@ const food = {
 };
 
 const tree = document.querySelector('#tree');
+const treeList = document.createElement('ul');
+
+tree.append(treeList);
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerHTML = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      const ul = document.createElement('ul');
+
+      li.append(ul);
+      createTree(ul, data[key]);
+    }
+
+    element.append(li);
+  }
```

Comment:
as I see, would be better to create `ul` in each call of `createTree` because it's new list

Reply:
I don't quite get an idea. Why we need to wrap our `li` into `ul` if we already put it inside `element` which is also `ul`